public/kibana-integrations/discover/application/angular/doc_table/components/table_row/_open.scss

```diff
@@ -12,3 +12,18 @@
   width: 14px; /* 1 */
   height: 14px;
 }
+
+.euiButtonIcon.euiButtonIcon--open {
```

Comment:
**thought**: are these classes exist in oui? If so, then this would be overwriting the style properties.

Reply:
Yes it exists, but it exists in the same file and that file is only being used in the Discover, so from Wazuh we can't access that class, and that class doesn't expand to Opensearch Dashboard

Reply:
I was reviewing with @yenienserrano and it seems the OpenSearch Dashboards 2.4.1 is doing the same thing https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.4.1/src/plugins/discover/public/application/angular/doc_table/components/table_row/_open.scss#L27-L40.